packages/e2e-test-app/metro.config.js

```diff
@@ -43,12 +43,14 @@ module.exports = {
       '@react-native-windows/virtualized-list': virtualizedListPath,
```

Comment:
@acoates-ms / @NickGerleman : Should we consider having a base metro.config.js file in react-native-windows and 'chain' our templates (or at least our samples here) to that configuration?

Reply:
Probably not a bad idea. -- but also would be good to get this in, the intermittent build failures this fixes will be great to get in. -- Probably want to port it to the 0.66 branch too.